tasks/config.yaml

```diff
@@ -29,6 +29,8 @@
       dest: "{{ nginx.prefix.config }}/include/proxy_headers.conf"
     - src: include/security_headers.conf
       dest: "{{ nginx.prefix.config }}/include/security_headers.conf"
+    - src: include/security_txt.conf
```

Comment:
I would split it into a separate `ansible.builtin.template` task with `when: nginx.security_txt.contact` as a running condition, instead of putting the condition in the conf. file itself. That way, we can avoid deploying an empty file if the security.txt info isn't filled.